lib/sprinkler_web/live/temperature_live.ex

```diff
@@ -0,0 +1,18 @@
+defmodule SprinklerWeb.TemperatureLive do
+  use Phoenix.LiveView
```

Comment:
Is it the same `use SprinklerWeb, :live_view` ?

Reply:
Nice catch, it’s not the same. It’s the equivalent of using `ApplicationRecord` vs `ActiveRecord::Model`